docs/reference/2-ods-api/how-to-guides/how-to-create-a-custom-populated-template.md

```diff
@@ -133,7 +105,7 @@ minutes to complete.
 #### Initialize-PopulatedTemplate Script Results
 
 ```powershell
-...
+<trimmed output...>
```

Comment:
This is how I dealt with code output where we are not showing the entire output. Hopefully this tells people that there was more output above, but we're only showing them the part we care about right now.

Reply:
Adding to migration notes..